cirq/experiments/fidelity_estimation.py

```diff
@@ -99,6 +99,41 @@ def log_xeb_fidelity_from_probabilities(
             np.mean(np.log(probabilities)))
 
 
+def hog_score_xeb_fidelity_from_probabilities(
+        hilbert_space_dimension: int,
+        probabilities: Sequence[float],
+) -> float:
+    """XEB fidelity estimator based on normalized HOG score.
```

Comment:
Add a reference that defines the HOG score.

Reply:
Done.